pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -131,6 +134,9 @@ type Route struct {
 	// MuteTimeIntervals is a list of MuteTimeInterval names that will mute this route when matched,
 	// +optional
```

Comment:
This one is not deprecated, since both `mute_time_intervals` and `active_time_intervals` are supported in the route. Added at https://github.com/prometheus-operator/prometheus-operator/pull/5135/files#diff-ca1bb1a59e5c40fa6d7fe5a21bba214234558c9ab699f3de0fdc95d7b302dc35R81 to mark `mute_time_intervals` in global config deprecated